doc/README.md

```diff
@@ -80,15 +80,27 @@ However, sending transactions in classic format is supported through the ARC API
 
 When possible, the API is responsible for rejecting transactions that would be unacceptable to the Bitcoin network.
 
+#### Callbacks
+
+The client can register to receive callbacks with information about the statuses of submitted transactions. To do this, the client must include the `X-CallbackUrl` header in their request. Once registered, the ARC will send a `POST` request to the URL specified in the header, with the transaction ID included in the request body.
+
+If the client wants to secure its callback endpoint, ARC supports Bearer token authorization. A callback token can be provided by adding the `X-CallbackToken: <your callback token>` header to the request.
+
+By default, callbacks are triggered when the submitted transaction reaches the status `REJECTED` or `MINED`. If the client wishes to receive additional intermediate status updates—such (e.g. `SEEN_IN_ORPHAN_MEMPOOL` or `SEEN_ON_NETWORK`) the `X-FullStatusUpdates` header must be set to true. For more details, refer to the [API documentation](https://bitcoin-sv.github.io/arc/api.html).
+
+The `X-MaxTimeout` header determines the maximum number of seconds the system will wait for new transaction statuses before the request expires. The default timeout is 5 seconds, with a maximum value of 30 seconds.
+
```

Comment:
I think it should be: 

"... wait for new transaction statuses before the response is returned."

I think this does not belong int the #### Callbacks section but rather under ### API